components/aws-comprehend/README.md

```diff
@@ -0,0 +1,54 @@
+This guide will walk you through how to train an AWS Comprehend machine learning classifier to classify email messages sent from this blueprint's Genesys Cloud email flow. AWS Cloudformation does not have support for the AWS Comprehend API at this point. For the training and deploying the classifier, you will need to use either the AWS management console or the AWS CLI (Command Line Interface) to train the classifier. For this blueprint, we will use the AWS CLI.
+
+# Pre-Requisites
+Before beginning this part of the tutorial please make sure you have done the following steps:
+
+1. Have a valid AWS account that is able to access and deploy AWS Comprehend. 
+2. Have a set of AWS credentials (eg. client id and secret). For more information on setting up your AWS credentials on your local machine see [here](https://docs.aws.amazon.com/sdkref/latest/guide/creds-config-files.html).
+3. Download and install the AWS CLI. For more information on installing the AWS CLI on your local machine see [here](https://aws.amazon.com/cli/).
+
+
+# Deployment Steps
+In order to train and deploy the classifier you need to take the following steps.
+
+**Note**: All AWS CLI commands are assumed to be run from the `components/aws-comprehend` directory.
+
+1. **Setup the S3 Bucket and copy the training corpus (`components/aws-comprehend/comprehendterm.csv`) to the created S3 bucket** 
+   `aws s3api create-bucket --acl private --bucket <<your-bucket-name-here>> --region <<Your region>>` 
+   
+   `aws s3 cp comprehendterms.csv s3://<<your-bucket-name-here>>`
+
+2. **Define the IAM Role and policy the AWS Comprehend Classifier will use.** 
+   `aws iam create-role --role-name EmailClassifierBucketAccessRole --assume-role-policy-document file://EmailClassifierBucketAccessRole-TrustPolicy.json`
+   
+   `aws iam put-role-policy --role-name EmailClassifierBucketAccessRole --policy-name BucketAccessPolicy --policy-document file://EmailClassifierBucketAccessRole-Permissions.json`
```

Comment:
> EmailClassifierBucketAccessRole

Good catch on the first one. Modified the file and fixed it.

Reply:
> I also encountered a challenge with directly attaching the role to the policy. The workaround I did is to create a policy first then attaching the created policy to the role.
> 
> `aws iam create-policy --policy-name BucketAccessPolicy --policy-document file://EmailClassifierBucketAccessRole-Permissions.json`
> 
> `aws iam attach-role-policy --policy-arn <<POLICY ARN HERE>> --role-name EmailClassifierBucketAccessRole`

Lets chat tomorrow.  Not quite following your comment.

Reply:
I updated the documentation here.  Can you please try to recreate the entire blueprint case and see if it looks good?  If everything is good please approve otherwise just drop me some more comments and I will keep fixing.  After you approve, I will put this through to